### ISSUE TITLE
Skin 10.6.0-0: textbox and menu-button default focus color should be blue

### ISSUE THREAD

**Issue by ushrivastava123** [eBay/skin#1174]
@ianmcburnie in skin 10.6.0-0, focus color is showing gray instead of blue for textbox and menu-button?
Below code might be the reason why:
```
.textbox {
    --textbox-background-color: @textbox-background-color;
    --textbox-foreground-color: @textbox-foreground-color;
    --textbox-border-color: @textbox-border-color;
    --textbox-focus-border-color: @textbox-border-color;
}
```
The textbox focus border color is same as textbox border color. Same is for menu-button. Please fix both of them before final release   "@ebay/skin": "^10.6.0-0".

**Comment by ianmcburnie** [comment]
@ushrivastava123 Seems to be just the textbox that is a bug. The menu button was never blue.

**Comment by ianmcburnie** [comment]
Fixed in 10.6.0 branch. Closing.
